testsuite/AdResS/HAdResS/test_HAdResS.py

```diff
@@ -0,0 +1,164 @@
+import sys
+import time
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+class TestHAdResS(unittest.TestCase):
+    def setUp(self):
+        # set up system
+        system = espressopp.System()
+        box = (10, 10, 10)
+        system.bc = espressopp.bc.OrthorhombicBC(system.rng, box)
+        system.skin = 0.3
+        system.comm = MPI.COMM_WORLD
+        nodeGrid = espressopp.tools.decomp.nodeGrid(espressopp.MPI.COMM_WORLD.size)
+        cellGrid = espressopp.tools.decomp.cellGrid(box, nodeGrid, 1.5, 0.3)
+        system.storage = espressopp.storage.DomainDecompositionAdress(system, nodeGrid, cellGrid)
+        self.system = system
+
+    def test_slab(self):
+        # add some particles
+        particle_list = [
+            (1, 1, 0, espressopp.Real3D(5.5, 5.0, 5.0), 1.0, 0),
+            (2, 1, 0, espressopp.Real3D(6.5, 5.0, 5.0), 1.0, 0),
+            (3, 1, 0, espressopp.Real3D(7.5, 5.0, 5.0), 1.0, 0),
+            (4, 1, 0, espressopp.Real3D(8.5, 5.0, 5.0), 1.0, 0),
+            (5, 1, 0, espressopp.Real3D(9.5, 5.0, 5.0), 1.0, 0),
+            (6, 0, 0, espressopp.Real3D(5.5, 5.0, 5.0), 1.0, 1),
+            (7, 0, 0, espressopp.Real3D(6.5, 5.0, 5.0), 1.0, 1),
+            (8, 0, 0, espressopp.Real3D(7.5, 5.0, 5.0), 1.0, 1),
+            (9, 0, 0, espressopp.Real3D(8.5, 5.0, 5.0), 1.0, 1),
+            (10, 0, 0, espressopp.Real3D(9.5, 5.0, 5.0), 1.0, 1),
+        ]
+        tuples = [(1,6),(2,7),(3,8),(4,9),(5,10)]
+        self.system.storage.addParticles(particle_list, 'id', 'type', 'q', 'pos', 'mass','adrat')
+        ftpl = espressopp.FixedTupleListAdress(self.system.storage)
+        ftpl.addTuples(tuples)
+        self.system.storage.setFixedTuplesAdress(ftpl)
+        self.system.storage.decompose()
+
+        # generate a verlet list
+        vl = espressopp.VerletListAdress(self.system, cutoff=1.5, adrcut=1.5,
+                                dEx=2.0, dHy=1.0, adrCenter=[5.0, 5.0, 5.0], sphereAdr=False)
+
+        # add interaction
+        interNB = espressopp.interaction.VerletListHadressLennardJones2(vl, ftpl)
+        potWCA1  = espressopp.interaction.LennardJones(epsilon=1.0, sigma=1.0, shift='auto', cutoff=1.4)
+        potWCA2 = espressopp.interaction.LennardJones(epsilon=0.0, sigma=1.0, shift='auto', cutoff=1.4)
+        interNB.setPotentialAT(type1=0, type2=0, potential=potWCA1) # AT
+        interNB.setPotentialCG(type1=0, type2=0, potential=potWCA2) # CG
```

Comment:
It's `type1=1, type2=1` (not that makes a difference in this particular test, but I noticed when I tried to copy and extend the test)